spec/test.go

```diff
@@ -27,7 +28,9 @@ func (t *Test) AddError(args Args, response *Response) error {
 }
 
 func main() {
+	socket_path := os.Args[1]
```

Comment:
Let's use the flags package here and make the default socket "/tmp/quartz.socket"